### ISSUE TITLE
Add subfieldLabelPosition to GQL schema

### ISSUE THREAD

**Issue by MangoMarcus** [verbb/formie#1797]
### What are you trying to do?

Support the "Subfield label position" on a headless site

### What's your proposed solution?

Add the `subfieldLabelPosition` field to the GQL schema

```gql
... on Field_Name {
  useMultipleFields
  subfieldLabelPosition
}
```

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-4 as 2.1.9"`.

**Comment by engram-design** [comment]
Fixed in [2.1.10](https://github.com/verbb/formie/releases/tag/2.1.10)
